README.md

```diff
@@ -109,14 +101,6 @@ Note that for all merges from master or stable, the merge should pick up:
 But never:
 * changes that bumped to a SHA inside config/components
 
-Here's a sample snippet used for a stable -> aardwolf merge:
```

Comment:
Thanks for jumping back in @pyther; this kinda slipped off my radar. I think you could just retain this stanza, but s/aardwolf/master/g it. Alternately, leave it in and someone else can do that bit :)